apps/ui/src/components/views/board-view/dialogs/pipeline-settings-dialog.tsx

```
@@ -702,10 +710,32 @@ export function PipelineSettingsDialog({
                     )
                   }
                   rows={6}
+                  rows={6}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)